core/txpool/celo_rates.go

```diff
@@ -0,0 +1,93 @@
+package txpool
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/contracts"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var (
+	unitRate = big.NewRat(1, 1)
+)
+
+func GetRatesForState(config *params.ChainConfig, state *state.StateDB) common.ExchangeRates {
+	stateCopy := state.Copy()
```

Comment:
Do we actually need this copy as the exchange rates are only read?

Reply:
forgot to remove this FN from the last refactor